### DISCUSSION TITLE
build构建时 会出现一堆的error

### DISCUSSION THREAD

**Discussion by ZhangKKKK** [youzan/vant#11189] [General]
### 重现链接

https://gitee.com/zhc007/recruit-h5

### Vant 版本

3.3.7

### 描述一下你遇到的问题。

node_modules/vant/lib/uploader/Uploader.d.ts:1:30 - error TS1005: ',' expected.

1 import { type PropType, type ExtractPropTypes } from 'vue';
                               ~~~~~~~~~~~~~~~~

node_modules/vant/lib/uploader/Uploader.d.ts:3:15 - error TS1005: ',' expected.

3 import { type ImagePreviewOptions } from '../image-preview';
                ~~~~~~~~~~~~~~~~~~~

Found 163 errors.

### 重现步骤

npm run build 

"build": "vue-tsc --noEmit && vite build --mode prod",

**Comment by ZhangKKKK** [comment]
![image](https://user-images.githubusercontent.com/25767148/198511945-88f399d1-47cd-4214-a7d9-7fc052d0f7f9.png)
npm run dev   dev: vite
启动之后报错

**Comment by ZhangKKKK** [comment]
> ```ts
> import type { PropType, ExtractPropTypes } from 'vue';
> import type { ImagePreviewOptions } from '../image-preview';
> ```

此报错为 执行build命令时 控制台出现的错误

**Comment by wjw-gavin** [comment]
你的仓库 看不了
![image](https://user-images.githubusercontent.com/19986739/198512286-a38daa27-a8ad-4564-8261-178dc4e1b68a.png)

**Comment by ZhangKKKK** [comment]
好了  麻烦down一下  之后继续关掉

**Comment by wjw-gavin** [comment]
> ![image](https://user-images.githubusercontent.com/25767148/198511945-88f399d1-47cd-4214-a7d9-7fc052d0f7f9.png) npm run dev dev: vite 启动之后报错

<img width="738" alt="image" src="https://user-images.githubusercontent.com/19986739/198513398-f42c6236-2f48-4947-85b8-e455327dfa0c.png">
我这正常的

**Comment by ZhangKKKK** [comment]
把router文件夹中 permission里边的login()注释掉     打开浏览器控制台  ，看看有没有报错， 再然后 执行一下build看看

**Comment by wjw-gavin** [comment]
build 报错先把 `"build": "vue-tsc --noEmit && vite build"`改成 `"build": "vite build"`吧

**Comment by Joe0926** [comment]
vue-tsc版本低了，升级一下就行
